reactioncommerce/reaction
# docs / style: Adds Styles for API Docs to match live Docs Styling

Resolves @reactioncommerce/reaction-static#97
Impact: **minor**  
Type: **style**

## Issue
[API Docs generated by JSdoc](http://api.docs.reactioncommerce.com/) command had outdated styling. This PR minimally edits the styles to make them match the live docs.reactioncommerce.com and adds the fonts directory to the JSDoc files. 

## Solution
Generated files now look like this:

![image](https://user-images.githubusercontent.com/779882/45129804-e65c2300-b153-11e8-832b-09baa44602b3.png)

## Breaking changes
No other jsdoc configuration changes were made, this is all presentation layer.

## Testing
1. Run `npm run docs`
2. cd into `/tmp/reaction-docs`
3. run `http-server .` (if you get an error, run `npm i http-server -g`)
4. visit `localhost:8080` to see generated docs.
